packages/next/src/build/webpack/loaders/next-metadata-image-loader.ts

```diff
@@ -30,15 +42,37 @@ async function nextMetadataImageLoader(this: any, content: Buffer) {
     type === 'favicon'
       ? ''
       : loaderUtils.interpolateName(this, '[contenthash]', opts)
+
   const outputPath =
     route + '/' + interpolatedName + (contentHash ? `?${contentHash}` : '')
 
-  let extension = loaderUtils.interpolateName(this, '[ext]', opts)
-  if (extension === 'jpg') {
-    extension = 'jpeg'
+  const isDynamicResource = pageExtensions.includes(extension)
+  if (isDynamicResource) {
+    // re-export and spread as `exportedImageData` to avoid non-exported error
+    return `\
```

Comment:
Unintended?

```suggestion
    return `
```

Reply:
Was intended to start the code in new line